### ISSUE TITLE
How to set N_rand value to get the same output as shown in the repository?

### ISSUE THREAD

**Issue by addy1997** [limacv/Deblur-NeRF#4]
Hi @limacv,

This is amazing work.

I am using Nvidia RTX 3060 GPU (Memory=VRAM 12 GB) to train the algorithm. 

After 1200 iterations, the code throws this error
![Screenshot 2022-07-01 12:10:09](https://user-images.githubusercontent.com/29406906/176883583-f1c8b86c-a8ba-4471-b85f-d5327b0bb300.png)

I was wondering how to set the `N_rand` value to get the output?

Thanks
Adwait

**Comment by addy1997** [comment]
I have one more query, is it possible to use image metrics like `Features Similarity Index Matrix (FSIM)` along with SSIM for this code?

**Comment by limacv** [comment]
Hi, I'm using 1 V100 (32GB) with N_rand = 1024. The memory required is roughly proportional to N_rand. So I guess you can first try N_rand = 1024 * (12 / 32) ~= 350. For other image metrics, just implement your own metrics in the test stage.

**Comment by addy1997** [comment]
Thanks for the response, @limacv. 

I was able to successfully (albeit with a smaller N_rand, chunk, and netchunk)  replicate the results of your algorithm on my GPU.

This is the command I used.

```bash
python3 run_nerf.py --config configs/demo_blurball.txt --N_rand 128 --chunk 384 --netchunk 768
```

1. Results after 20,000 iterations

https://user-images.githubusercontent.com/29406906/177553625-aad476e1-b1bd-4504-bd6a-9b7458824da3.mp4

2. Results after 40,000 iterations

https://user-images.githubusercontent.com/29406906/177553707-598db34f-23ee-494e-8343-ddc43310fba2.mp4

3. Results after 60,000 iterations (final)

https://user-images.githubusercontent.com/29406906/177553803-0ea441c2-d707-4828-b0c4-8cc31fc94107.mp4

Thanks for your help.
